### PR TITLE
Changing the caption for the GitHubTeacher slide

### PR DESCRIPTION
**Pull request by githubteacher**
Ask for review

Ask leading questions

Describe, in detail, the PR

Attach screenshots to demonstrate changes

### CODE REVIEW

File: _posts/0000-01-29-githubteacher.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Hollenberry's Slide"
 ---
 
 ![fresh-prince](https://cloud.githubusercontent.com/assets/16547949/25400882/f44a7464-29c1-11e7-8f45-327d67cb961a.jpg)
```
Comment (beardofedu) [inline _posts/0000-01-29-githubteacher.md:6]: Needs more Bob's!

lol

File: _posts/0000-01-29-githubteacher.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 {: .center}
 
-Caption here
+I wasn't born and raised in West Philadelphia
```
Comment (beardofedu) [inline _posts/0000-01-29-githubteacher.md:9]: if you change your image, your caption might need to be changed as well

**Review by beardofedu — CHANGES_REQUESTED** [review]

File: _posts/0000-01-29-githubteacher.md (hunk 2 above)
Comment (rafaelsaldana) [reply _posts/0000-01-29-githubteacher.md:9]: 💯

**Review by beardofedu — APPROVED** [review]
hurtful.

**Pull request merged**